src/code/audio_synthesis.c

```diff
@@ -1,25 +1,22 @@
 #include "ultra64.h"
 #include "global.h"
 
-#define DEFAULT_LEN_1CH 0x1A0
-#define DEFAULT_LEN_2CH 0x340
-
 // DMEM Addresses for the RSP
 #define DMEM_TEMP 0x3C0
 #define DMEM_UNCOMPRESSED_NOTE 0x580
 #define DMEM_NOTE_PAN_TEMP 0x5C0
-#define DMEM_SCRATCH2 0x760              // = DMEM_TEMP + DEFAULT_LEN_2CH + a bit more
+#define DMEM_SCRATCH2 0x760              // = DMEM_TEMP + DMEM_2CH_SIZE + a bit more
 #define DMEM_COMPRESSED_ADPCM_DATA 0x940 // = DMEM_LEFT_CH
 #define DMEM_LEFT_CH 0x940
 #define DMEM_RIGHT_CH 0xAE0
 #define DMEM_WET_TEMP 0x3E0
-#define DMEM_WET_SCRATCH 0x720 // = DMEM_WET_TEMP + DEFAULT_LEN_2CH
+#define DMEM_WET_SCRATCH 0x720 // = DMEM_WET_TEMP + DMEM_2CH_SIZE
```

Comment:
What's the reason to not use that comment for the define?

Reply:
A combination of not looking at dmem addresses yet, but also deciding if there's actual logic to this or not. I think that's a separate PR at least